Makefile

```diff
@@ -1,5 +1,6 @@
 include experiment.mk
 
+DO_INSTANCE_TAGNAME=main-testnet
 DO_VPC_SUBNET=172.19.144.0/20
```

Comment:
The idea to move these two variables together is that it's necessary to customize them both to deconflict QA testnet experiments running concurrently in the same project. If `experiment.mk` looks like a more appropriate place for this, let's move them both there.